onboard/catkin_ws/src/offboard_comms/launch/serial.launch

```diff
@@ -0,0 +1,8 @@
+<launch>
```

Comment:
Add the servo_wrapper.py to this launch file, might as well have it integrated with the rest of the system.

Also, add this launch file to one of the launch files in the execute package (maybe motion.launch?). Make sure to not run this when sim=True, however you wish to handle that.